checker/src/org/checkerframework/checker/experimental/regex_qual/RegexQualifiedTransfer.java

```diff
@@ -82,6 +82,8 @@ public RegexQualifiedTransfer(QualAnalysis<Regex> analysis) {
                         thenStore.insertValue(firstParam, new RegexVal(0));
                     }
                 } catch (FlowExpressionParseException e) {
+                    // report errors here
+                    // checker.report(e.getResult(), n);
```

Comment:
Ah, sorry I missed this. Do you know the proper way to report errors in the Qual Framework? I found a way to do it, but it clearly isn't the intended design:
`                analysis.getContext().getChecker().getCheckerAdapter().getVisitor().getTypeFactory().getContext().getChecker().report(e.getResult(), n);`